localization/english/dvg_flavor_venice_l_english.yml

```diff
@@ -50,4 +48,404 @@ Zimisces: "Zimisces"
+Premarin: "Premarin"
+Sagredo: "Sagredo"
+Selvo: "Selvo"
+Semitecolo: "Semitecolo"
+Steno: "Steno"
+Storladi: "Storladi"
+Stornello: "Stornello"
+Surian: "Surian"
+Valaresso: "Valaresso"
+Valier: "Valier"
+Vitturi: "Vitturi"
+Zancaruol: "Zancaruol"
+Zantani: "Zantani"
+Zulian: "Zulian"
+Albizzo: "Albizzo"
+Basadonna: "Basadonna"
+Coppo: "Coppo"
+dalle_Boccole: "dalle Boccole"
+da_Lezze: "da Lezze"
+d'Arduin: "d'Arduin"
+Fabriciacio: "Fabriciacio"
+Galanti: "Galanti"
+Gambarin: "Gambarin"
+Lanzuoli: "Lanzuoli"
+Lombardo: "Lombardo"
+Mazaman: "Mazaman"
+Miegano: "Miegano"
+Mussolino: "Mussolino"
+Navigroso: "Navigroso"
+Sesendillo: "Sesendillo"
+Signolo: "Signolo"
+Viaro: "Viaro"
+Vielmo: "Vielmo"
+Volpe: "Volpe"
+Zaguri: "Zaguri"
+Zancani: "Zancani"
+Acotanto: "Acotanto"
+Bonomo: "Bonomo"
+Mastalizi: "Mastalizi"
+Ruzzini: "Ruzzini"
+Agadi: "Agadi"
+Caotorta: "Caotorta"
+Quintavalle: "Quintavalle"
+Addoldo: "Addoldo"
+Agrinal: "Agrinal"
+Buoninsegna: "Buoninsegna"
+Caroso: "Caroso"
+Diente: "Diente"
+Diesello: "Diesello"
+Ferro: "Ferro"
+Grisoni: "Grisoni"
+Papaciza: "Papaciza"
+Vidor: "Vidor"
+Albrizzi: "Albrizzi"
+Angaran: "Angaran"
+Antelmi: "Antelmi"
+Ariberti: "Ariberti"
+Barbaran: "Barbaran"
+Belloni: "Belloni"
+Beregan: "Beregan"
+Berlendis: "Berlendis"
+Bonfadini: "Bonfadini"
+Bonlini: "Bonlini"
+Bonvicini: "Bonvicini"
+Bregonzi: "Bregonzi"
+Bressa: "Bressa"
+Cassetti: "Cassetti"
+Catti: "Catti"
+Conti: "Conti"
+Correggio: "Correggio" 
+Crotta: "Crotta"
+Dolce: "Dolce"
+Farsetti: "Farsetti" 
+Ferramosca: "Ferramosca" 
+Ferro: "Ferro"
+Fini: "Fini"
+Flangini: "Flangini"
+#Is the light near, I can no longer see it
+Fonseca: "Fonseca"
+Fonte: "Fonte"
+Gambara: "Gambara"
+Ghedini: "Ghedini"
+Gheradini: "Gheradini"
+Giovanelli: "Giovanelli"
+Giupponi: "Giupponi"
+Gozzi: "Gozzi"
+Labia: "Labia"
+Laghi: "Laghi"
+Lazzari: "Lazzari"
+Lombria: "Lombria" 
+Lucca: "Lucca" 
+Macarelli: "Macarelli" 
+Maffetti_Manin: "Maffetti Manin" 
+Martinelli: "Martinelli" 
+Medici: "Medici"
+Minelli: "Minelli" 
+Mora: "Mora"
+Nave: "Nave"
+Ottoboni: "Ottoboni"
+Papafava: "Papafava"
+Pasta: "Pasta"
+Piovene: "Piovene" 
+Poli: "Poli" 
+Polvaro: "Polvaro"
+Raspi: "Raspi"
+Ravagnin: "Ravagnin"
+Rubini: "Rubini" 
+Sangiantoffetti: "Sangiantoffetti"
+Santasoffia: "Santasoffia"
+Soderini: "Soderini" 
+Statio: "Statio" 
+Surian: "Surian"
+Tasca: "Tasca"
+Valmarana: "Valmarana" 
+Verdizzotti: "Verdizzotti" 
+Vianol: "Vianol" 
+Zacco: "Zacco" 
+Zaguri: "Zaguri" 
+Zambelli: "Zambelli" 
+Zanardi: "Zanardi" 
+Zenobio: "Zenobio" 
+Zolio: "Zolio" 
+Zon: "Zon"
+Acquisti: "Acquisti" 
+Arnaldi: "Arnaldi" 
+Baglioni: "Baglioni" 
+Barzizza: "Barzizza" 
+Bellotto: "Bellotto" 
+Benzon_di_San_Vidal: "Benzon di San Vidal" 
+Brandolini: "Brandolini" 
+Carminati: "Carminati" 
+Castelli: "Castelli" 
+Cavagnis: "Cavagnis" 
+Celini: "Celini" 
+Codognola: "Codognola" 
+Contenti: "Contenti" 
+Cottoni: "Cottoni" 
+Curti: "Curti" 
+Fracassetti: "Fracassetti" 
+Franceschi: Franceschi" 
```

Comment:
You are missing a " here

Reply:
I hope that fixes it. I really want to keep the Venice names since I took way too long in fixing and updating them

Reply:
They are still missing localization...
And also this error:
[18:34:27][localization_reader.cpp:135]: Expected colon(:) at line 199 and column 4 in localization/english/dvg_flavor_venice_l_english.yml
[18:34:27][localization_reader.cpp:135]: Expected colon(:) at line 208 and column 2 in localization/english/dvg_flavor_venice_l_english.yml
[18:34:27][localization_reader.cpp:135]: Expected colon(:) at line 227 and column 5 in localization/english/dvg_flavor_venice_l_english.yml
[18:34:27][localization_reader.cpp:135]: Expected colon(:) at line 266 and column 6 in localization/english/dvg_flavor_venice_l_english.yml

Reply:
I'm confused how they are missing localization 
Most of the dvg_flavor_venice_l_english.yml is just localization of the names...
Also I'll just delete the error lines, there probably a funky name